src/ambianic/webapp/flaskr.py

```diff
@@ -159,6 +161,66 @@ def get_status():
         resp = jsonify(response_object)
         return resp
 
+    # start the auth0 authentication process with creating the auth0 keys
```

Comment:
@vickywane these api token management rest api methods make sense, but I need to see them in a more detailed UML sequence diagram to understand exactly how they will fit in the PWA UX flow.

Reply:
The existing UML diagram has been updated and referenced on Slack

Reply:
It needs to be referenced here, in the issue that it solves and most importantly in the project DOCS!! Slack is not a searchable web resource. It's only useful for short lived chats.

Reply:
I have updated this Pull request description to contain the document link for better visibility. [This](https://app.diagrams.net/#G1gDYMs2LjbZ9dnJG8i5yl5UFMGAJ1nABq) is also the link.

The `readme` within this PR would also be updated to include the new Notifications

Reply:
Link leads to inaccessible
![Screen Shot 2021-04-05 at 8 44 00 AM](https://user-images.githubusercontent.com/2234901/113580291-2117e780-95eb-11eb-92e1-c0920dcf17f9.png)
 file.

Reply:
Make sure to embed the diagram view in the comments AND verify access with anonymous browser to save us back and forth on these access issues.